packages/ember-routing/lib/system/route.js

```diff
@@ -1762,7 +1762,7 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
   },
 
   /**
-    Returns the resolved model of the current route, or a parent (or any ancestor)
+    Returns the resolved controller of the current route, or a parent (or any ancestor)
```

Comment:
I realize this isn't really related to your change, but we should probably remove `resolved` from here.  We don't do any promise resolution, and it looks like this was just copy/pasta'ed from `modelFor` (which does resolve promisey things).

**tldr;** Would you mind removing `resolved` here?